packages/integration_test/lib/integration_test.dart

```diff
@@ -20,24 +20,35 @@ import 'common.dart';
 
 const String _success = 'success';
 
+/// Whether results should be reported to the native side over the method
+/// channel.
+///
+/// This is enabled by default for use by native test frameworks like Android
+/// instrumentation or XCTest. When running with the Flutter Tool through
+/// `flutter test integration_test` though, it will be disabled as the Flutter
+/// tool will be responsible for collection of test results.
+const bool _shouldReportResultsToNative = bool.fromEnvironment(
+  'INTEGRATION_TEST_SHOULD_REPORT_RESULTS_TO_NATIVE',
+  defaultValue: true,
+);
+
 /// A subclass of [LiveTestWidgetsFlutterBinding] that reports tests results
 /// on a channel to adapt them to native instrumentation test format.
 class IntegrationTestWidgetsFlutterBinding extends LiveTestWidgetsFlutterBinding implements IntegrationTestResults {
   /// Sets up a listener to report that the tests are finished when everything is
   /// torn down.
   IntegrationTestWidgetsFlutterBinding() {
-    // TODO(jackson): Report test results as they arrive
```

Comment:
Did you eman to remove this?

Reply:
Yes, I think this is obsolete now. We're already doing so when running with `flutter test`, and I don't think it's on our radar to do so for platform native reporting.